src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -353,6 +355,8 @@ def launch_fluent(
             mode = LaunchModes.get_mode(mode)
         new_session = mode.value[1]
         meshing_mode = mode.value[2]
+        for k, v in mode.value[3]:
```

Comment:
Apply the last item in the session definition tuple, to the launcher command line environment
SOLVER_ICING = ("solver-icing", SolverIcing, False, **[("fluent_icing", True)]**)

will end up binding to "-flicing" fluent command line